tests/test_eval.py

```diff
@@ -15,7 +15,10 @@
     ({"=": 3, "ignore": 5}, 3),
     ({"=sum": [3, 5]}, 8),
     ({"=sum": (3, 5)}, 8),
-    ({"=sum": {"=": [3, 5]}}, 8)
+    ({"=sum": {"=": [3, 5]}}, 8),
+    ({"=mul": [3, 5]}, 15),
+    ({"=mul": (3, 5)}, 15),
```

Comment:
The `=sum` already tests for different sequences like `list`/`tuple`. I think you can remove this test. I'd like to see a test with more than 2 elements in list...